cloudfunctions/candidate_gmail_messages/cloudfunction.go

```diff
@@ -344,11 +350,67 @@ func (cf *CloudFunction) proccessMessages(messageIDs []string) error {
 
 	// Get IDs of new recruiting emails
 	recruitingEmailIDs := []string{}
-	for id, result := range results.Results {
-		if !result {
+	for id, isRecruitingEmail := range results.Results {
+		// ignore non-recruting emails based on classification
+		if !isRecruitingEmail {
 			continue
 		}
 		recruitingEmailIDs = append(recruitingEmailIDs, id)
+		message := messages[id]
+		parseRequest := ml.ParseJobRequest{
+			From:    srcmessage.Sender(message),
+			Subject: srcmessage.Subject(message),
+			Body:    srcmessage.Body(message),
+		}
+
+		log.Printf("parsing email: %s", message.Id)
+		job, err := cf.model.ParseJob(&parseRequest)
+		// for now, abort on error
+		if err != nil {
+			return err
+		}
+
+		// if fields are missing, skip
+		if job.Company == "" || job.Title == "" || job.Recruiter == "" {
+			// print sender and subject
+			log.Printf("skipping job: %v", job)
+			continue
+		}
+		recruiterEmail := srcmessage.SenderEmail(message)
+		data := map[string]interface{}{
+			"recruiter":       job.Recruiter,
+			"recruiter_email": recruiterEmail,
+		}
+
+		// turn data into json.RawMessage
+		b, err := json.Marshal(data)
+		if err != nil {
+			return err
+		}
+
+		// convert epoch ms to time.Time
+		emailedAt := time.Unix(message.InternalDate/1000, 0)
```

Comment:
Can you update this to use this new helper `srcmessage.CreatedAt`?
```suggestion
		emailedAt := srcmessage.CreatedAt(message)
```